_posts/2019-09-18-cockpit-203.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Cockpit 203
+author: mvo
+date: 2019-09-18
+tags: cockpit linux
+slug: cockpit-203
+category: release
+summary: Cockpit with Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 203.
+
+### Machines: Network Interfaces
+
+It is now possible to add network interfaces to virtual machines.
```

Comment:
```suggestion
Virtual machines now support network interface selection.
```

Reply:
@garrett, afaik, this is not about selecting interfaces but actually creating new ones.  Like buying a new network card, but it's all virtual.

Reply:
I have changed it to "Virtual machines now support creation of virtual network interfaces."